src/bmi_cfe.c

```diff
@@ -583,7 +583,7 @@ int read_init_config_cfe(const char* config_file, cfe_state_struct* model)
             }
             continue;
         }
-        if (strcmp(param_key, "soil_params.smcmax") == 0 || strcmp(param_key, "soil_params.maxsmc") == 0) {
+        if (strcmp(param_key, "maxsmc") == 0 || strcmp(param_key, "soil_params.smcmax") == 0 || strcmp(param_key, "soil_params.maxsmc") == 0) {
```

Comment:
we have `maxsmc` but not `smcmax`, they are used interchangeably, so better to add it as well

Reply:
Yeah, I only added `maxsmc` because that is the name of the calibratable parameter. I thought including a short form name would be more confusing. I guess alternatively we could change the calibratable parameter name to `smcmax`?